src/main/resources/application.properties

```diff
@@ -1 +1,4 @@
-
+spring.jpa.hibernate.ddl-auto=none
+spring.datasource.url=jdbc:mysql://localhost:3306/gas_calc
+spring.datasource.username=admin
+spring.datasource.password=IT4045
```

Comment:
This is a basic database connection to a table matching the description I was given. This will allow you to start working on the persistence layer, as you will be able to access data from your MySql table.